lib/entryBrowser.ts

```diff
@@ -1,3 +1,4 @@
 import AlgoliaAnalytics from "./insights";
 
-export default new AlgoliaAnalytics();
+const processQueue = typeof window !== "undefined";
+export default new AlgoliaAnalytics({ processQueue });
```

Comment:
I'm not sure it makes sense to have an option dedicated to this behaviour. We could instead run the `processQueue` function only inside the `browser` module. We avoid the introduction of an option only for this purpose, which makes the `insights` a bit more platform agnostic.

Reply:
Okay, since it's executed right after instantiation, we could just call `processQueue` directly to the instance at `browser` module.
👍

Reply:
5f93971